src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -123,6 +123,28 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
                 result = SetResolveData(result.Value, serverKind);
             }
 
+            // This is a bit fraught. Basically in the scenario That someone types "@<slight pause>{" they'll get @__builder because { is a completion item.
```

Comment:
```suggestion
            // This is a bit fraught. Basically in the scenario That someone types "@<slight pause>{" they'll get @__builder because `{` is a commit character.
```